elftools/elf/structs.py

```diff
@@ -153,6 +154,16 @@ def _create_shdr(self):
             self.Elf_xword('sh_entsize'),
         )
 
+    def _create_chdr(self):
+        fields = [
```

Comment:
I used my distrib’s /usr/include/elf.h, which seems to come from glib: https://sourceware.org/git/?p=glibc.git;a=blob;f=elf/elf.h;h=3900b4c9f0cad606a7beb2511c0bb6d603e885ee;hb=HEAD#l489 Is this good enough? I can also find http://www.sco.com/developers/gabi/latest/ch4.sheader.html, but there’s no anchor link for `SHF_COMPRESSED` and I’m not sure about the perenity of the URL.